lib/ruby_lsp/listeners/definition.rb

```diff
@@ -145,6 +147,22 @@ def handle_require_definition(node)
         end
       end
 
+      sig { params(node: Prism::CallNode).void }
+      def handle_autoload_definition(node)
+        name = node.name # should be ":autoload"
+        if name != :autoload
+          return
+        end
```

Comment:
This is already being checked where the method is invoked, so this can be removed.

Reply:
got it